bin/codemods/run.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
+	const excludeUtilityFiles = name => name !== 'config.js' && name !== 'helpers.js';
+	const excludeDotFiles = name => name[0] !== '.';
+	const includeJSFiles = name => path.extname( name ) === '.js';
+	return fs.readdirSync( './bin/codemods/src' ).filter(
+		excludeUtilityFiles
+	).filter(
+		excludeDotFiles
+	).filter(
+		includeJSFiles
+	).map( name => name.replace( '.js', '' ) );
+
+}
+
+function getValidCodemodNames() {
+	return [
+		...getLocalCodemodFileNames(),
+		...Object.getOwnPropertyNames( config.codemodArgs ),
+	].map( name => '- ' + name ).sort();
+}
+
+function printableValidCodemodNames() {
+	return `${ getValidCodemodNames().join('\n') }`
+}
+
+function generateBinArgs( name ) {
+	if ( config.codemodArgs.hasOwnProperty( name ) ) {
+		// Is the codemod defined in the codemodArgs object?
+		return config.codemodArgs[ name ];
+	} else if ( getLocalCodemodFileNames().includes( name ) ) {
+		// Is the codemod a local script defined in bin/codemods/src folder?
+		return [
+			`--transform=bin/codemods/src/${ name }.js`,
+		];
+	} else {
+		throw new Error(
+			`"${ name }" is an unrecognized codemod.`
+		);
+	}
```

Comment:
we do tend to prefer early-abort over `else if`s

```js
if ( hasProp( name ) ) {
	return config.…
}

if ( getNames().includes( … ) ) {
	return …
}

throw new Error( … )
```

Reply:
Fixed!